src/test/java/com/criptext/mail/scenes/signup/data/AccountDataValidatorTest.kt

```diff
@@ -15,47 +15,55 @@ class AccountDataValidatorTest {
     fun `validateUsername should return valid if receives a valid username`() {
         val myUsername = "tester"
         AccountDataValidator.validateUsername(myUsername) `should equal` FormData.Valid("tester")
+        AccountDataValidator.validateUsernameOnly(myUsername) `should equal` FormData.Valid("tester")
     }
 
     @Test
     fun `validateUsername should return valid if receives less than 3 characters`() {
         val myUsername = "t"
         AccountDataValidator.validateUsername(myUsername) `should be instance of` FormData.Error::class.java
+        AccountDataValidator.validateUsernameOnly(myUsername) `should be instance of` FormData.Error::class.java
     }
 
     @Test
     fun `validateUsername should return error if receives spaces in the middle of the username`() {
         val myUsername = "anew tester"
         AccountDataValidator.validateUsername(myUsername) `should be instance of` FormData.Error::class.java
+        AccountDataValidator.validateUsernameOnly(myUsername) `should be instance of` FormData.Error::class.java
     }
 
     @Test
     fun `validateUsername should return error if receives invalid characters in username`() {
         val myUsername = "test#"
         AccountDataValidator.validateUsername(myUsername) `should be instance of` FormData.Error::class.java
+        AccountDataValidator.validateUsernameOnly(myUsername) `should be instance of` FormData.Error::class.java
     }
 
     @Test
     fun `validateUsername should return error if username starts with dot`() {
         val myUsername = ".tester"
         AccountDataValidator.validateUsername(myUsername) `should be instance of` FormData.Error::class.java
+        AccountDataValidator.validateUsernameOnly(myUsername) `should be instance of` FormData.Error::class.java
     }
 
     @Test
     fun `validateUsername should return valid if receives dots, dashes or lowdashes in username`() {
         val myUsername = "test.t_est-er05"
         AccountDataValidator.validateUsername(myUsername) `should be instance of` FormData.Valid::class.java
+        AccountDataValidator.validateUsernameOnly(myUsername) `should be instance of` FormData.Valid::class.java
     }
 
     @Test
     fun `validateEmailAddress should return valid if receives a valid address`() {
         val address = "somebody@gmail.com"
         AccountDataValidator.validateEmailAddress(address) `should equal` FormData.Valid(address)
+        AccountDataValidator.validateUsername(address) `should equal` FormData.Valid(address)
```

Comment:
I would create another test where `validateUsernameOnly` fails when an email is passed